### PR TITLE
Batch size 8 + scaled Lion LR (5e-5): optimization stability for wall-shear gap

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Doubling batch size from 4 to 8 with a proportionally scaled Lion learning rate (5e-5, applying the linear scaling rule: lr ∝ batch_size) will stabilize gradient estimates, reduce per-step noise in τ_y/τ_z optimization, and improve wall-shear metrics. Larger batches provide more representative gradient directions per step, which may help Lion navigate the anisotropic loss landscape that leaves τ_y and τ_z 2.6–3× above the AB-UPT reference targets. This configuration (bs=8) was used in early protocol runs (PR #9 gilbert) but has never been tested on the mature SOTA stack (learnable-PE + grad-ema + beta-NLL + k1_k2 curvature).

## Instructions

Run a cold-start training from scratch on the full yi SOTA config with batch_size=8 and lr=5e-5. Use 4 GPUs with per-device batch=2 (effective global batch=8). All other hyperparameters identical to the yi SOTA cold-start.

**Reproduce command:**
```bash
cd target/
python train.py --help  # verify flags
CUDA_VISIBLE_DEVICES=0,1,2,3 torchrun --standalone --nproc_per_node=4 train.py \
  --agent armin \
  --wandb-group yi-round38-batch8-lr5e5 \
  --wandb-name "armin/batch8-lr5e5-cold-start" \
  --learnable-pe \
  --optimizer lion \
  --lr 5e-5 \
  --weight-decay 5e-4 \
  --clip-grad-norm 0.5 \
  --grad-ema-alpha 0.5 \
  --ema-decay 0.999 \
  --model-layers 4 \
  --model-hidden-dim 512 \
  --model-heads 8 \
  --model-slices 128 \
  --batch-size 8 \
  --beta-nll-beta 0.5 \
  --surface-curvature-features k1_k2 \
  --train-surface-points 65536 \
  --eval-surface-points 65536 \
  --train-volume-points 65536 \
  --eval-volume-points 65536 \
  --validation-every 1 \
  --no-compile-model \
  --lr-warmup-epochs 1 \
  --gradient-log-every 100 \
  --weight-log-every 100
```

Note: `--batch-size 8` with `--nproc_per_node=4` gives per-device batch of 2. Effective batch = 8. The `--lr 5e-5` is derived by applying the linear scaling rule from the SOTA lr=1e-4 at bs=4 → halved to 5e-5 at bs=8.

Run until SENPAI_TIMEOUT_MINUTES. Report val_abupt and all 7 sub-metrics at the best validation checkpoint. If training is stable and metrics are improving at timeout, note the val trajectory.

## Baseline

Current yi SOTA: **PR #658** (nezuko, EMA best-ckpt from lr=5e-6 SWA run), W&B run `pxsnrw36`

| Metric | val | test |
|---|---:|---:|
| abupt_axis_mean_rel_l2_pct | **7.3914%** | 8.7189% |
| surface_pressure_rel_l2_pct | 4.8552% | — |
| wall_shear_rel_l2_pct | 8.3192% | — |
| volume_pressure_rel_l2_pct | 4.3156% | ~11.46% |
| wall_shear_x_rel_l2_pct (τ_x) | 7.1166% | — |
| wall_shear_y_rel_l2_pct (τ_y) | 9.6123% | 9.7675% |
| wall_shear_z_rel_l2_pct (τ_z) | 11.0573% | 10.8851% |

**AB-UPT reference targets (test):** surface_pressure 3.82%, wall_shear 7.29%, volume_pressure 6.08%, τ_x 5.35%, τ_y 3.65%, τ_z 3.63%

To beat: val_abupt < 7.3914%

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Kicking off your experiment. A few important corrections before you start:

**Merge bar correction**: The baseline in the PR body is stale. PR #681 (nezuko, terminal LR polish lr=3e-7) has since been merged and is the new SOTA. The correct target is **val_abupt < 7.3767%** (not 7.3914%). W&B run `dc031qpt`. Updated full baseline:

| Metric | val | test |
|---|---:|---:|
| abupt_axis_mean_rel_l2_pct | **7.3767%** | **8.7015%** |
| surface_pressure_rel_l2_pct | 4.8515% | 4.6236% |
| wall_shear_rel_l2_pct | 8.3016% | 8.3214% |
| volume_pressure_rel_l2_pct | 4.3066% | 11.3738% |
| wall_shear_x_rel_l2_pct (τ_x) | 7.1045% | 7.1753% |
| wall_shear_y_rel_l2_pct (τ_y) | 9.5832% | 9.5964% |
| wall_shear_z_rel_l2_pct (τ_z) | 11.0377% | 10.7383% |

**Kill-threshold correction**: Use `val_abupt < 50` (less-than), NOT `>= 50`. The correct gate is: kill the run if val_abupt is **not** below 50% at EP1. In practice for a healthy cold-start, expect EP1 val_abupt in the range of 18–30%; if it's above 50% the run has diverged.

**4-GPU execution**: Run sequentially on all 4 GPUs as specified (`--nproc_per_node=4`). Do not run on fewer GPUs.

**EP1 gate**: After epoch 1 completes, report val_abupt. Expected healthy range for this batch-size=8/lr=5e-5 config: ~20–30% (cold starts typically start higher, then descend steeply). If EP1 > 50%, kill the run and check for a configuration error. If EP1 is in the 20–30% range, continue.

**What to watch**: With bs=8 (per-device=2, global=8) and lr=5e-5, you're running at half the LR of the SOTA config (linear scaling from bs=4/lr=1e-4). The key question is whether the larger batch stabilizes τ_y/τ_z gradient estimates. Report the wall-shear sub-metrics at every validation epoch.

Please start the run and post an update after EP1. Good luck!

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #683 because ADVISOR: Closing orphaned draft. Student armin is no longer in the active 16-student lineup; this draft assignment was never started..
